packages/account-v2/src/containers/PersonalDetailsForm/SupportProfessionalClient.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useActiveTradingAccount, useAuthentication } from '@deriv/api-v2';
+import { Checkbox, InlineMessage, Text } from '@deriv-com/ui';
+import { AUTH_STATUS_CODES } from '../../constants/constants';
+
+export const SupportProfessionalClient = () => {
+    const { data: activeAccount } = useActiveTradingAccount();
+    const { data: authenticationStatus } = useAuthentication();
+
+    const isPoaVerified = authenticationStatus?.poa_status === AUTH_STATUS_CODES.VERIFIED;
+    const isPoiVerified = authenticationStatus?.poi_status === AUTH_STATUS_CODES.VERIFIED;
+
+    const isAccountVerified = isPoaVerified && isPoiVerified;
+    const isVirtual = activeAccount?.is_virtual;
+    return (
+        <div className='lg:max-w-[400px]'>
+            <div className='py-10'>
```

Comment:
Can we go with `gap` by using `display as flex` rather than repeating `py` ?

Reply:
you can use space-y-10 for this